### DISCUSSION TITLE
Advanced Usage wiki shows usage for removed classes in ShardedJedis section

### DISCUSSION THREAD

**Discussion by david-wiles** [xetorthio/jedis#3513] [Q&A] [answered]
As mentioned in the title, the information in https://github.com/redis/jedis/wiki/AdvancedUsage#shardedjedis is outdated. 

What is the recommended way to use Redis when hosts are used as a sharded 'cluster' now? Can I replace `ShardedJedisPool` with `JedisSharding` as mentioned in https://github.com/redis/jedis/blob/master/docs/3to4.md, specifically for a multithreaded environment?

**Comment by sazzad16** [comment, marked as answer]
As of Jedis 5, there is no recommended way. The complete Sharding/Sharded feature is [deprecated](https://github.com/redis/jedis/pull/3386).

In Jedis 4, you can use JedisSharding and it does support multithreaded environment.

**Reply by david-wiles** [reply]
👍 Thank you!
